src/modules/hub.bicep

```diff
@@ -5,6 +5,11 @@
 @description('Optional. Name of the hub. Used to ensure unique resource names. Default: "finops-hub".')
 param hubName string
 
+@minLength(3)
+@maxLength(63)
+@description('Optional. Name of the data factory. Used to ensure unique resource names. Default: "finops-hub-engine".')
+param dataFactoryName string
```

Comment:
Do we need to take this as an input? I'd rather have fewer inputs and standardize the name across implementations. If they change it, then there's more risk that they might misconfigure it when they deploy the next release. Ultimately, when this is a managed solution, this will be a bit of a black box that they won't be able to change. Do you have any concerns with that approach?